lms/djangoapps/ccx/urls.py

```diff
@@ -25,6 +25,8 @@
     url(r'^ccx_gradebook/(?P<offset>[0-9]+)$',
         'ccx.views.ccx_gradebook', name='ccx_gradebook'),
 
+    url(r'^rename_ccx$', 'ccx.views.rename_ccx', name='rename_ccx'),
```

Comment:
with a patch this will be much easier, but we need to wait on that PR to be merged to merge this one: I'm not sure if we want to start to introduce this dependency.
In general I think we should move the entire CCX UI to use the REST APIs for these things and extend the REST API for whatever we are missing (like the ccx schedule, CCX Enrollment, etc...)